extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -250,8 +250,8 @@ export class AjaxMonitor extends BaseTelemetryPlugin implements IDependenciesPlu
                 _self.setInitialized(false);
             }
 
-            _self.trackDependencyData = (dependency: IDependencyTelemetry, properties?: { [key: string]: any }) => {
-                _trackDependencyDataInternal(dependency, properties);
+            _self.trackDependencyData = (dependency: IDependencyTelemetry, properties?: { [key: string]: any }, systemProperties?: { [key: string]: any }) => {
```

Comment:
While systemProperties is not used directly by app insights is passed by an external extension plugin